js_modules/dagster-ui/packages/ui-core/src/workspace/WorkspaceContext.tsx

```diff
@@ -171,10 +172,12 @@ export const ROOT_WORKSPACE_QUERY = gql`
       status
     }
     sensorType
+    ...SensorSwitchFragment
```

Comment:
I thought I remembered this being expensive. This will now be retrieved at the workspace root, on every pageload. Is that acceptable?

Reply:
The workspace query assumes that it has sensor switch fragments, regardless of whether it actually uses the SensorSwitchFragment fragment or not.